app/views/label_formats/index.html.haml

```diff
@@ -0,0 +1,45 @@
+-#  Copyright (c) 2012-2013, Jungwacht Blauring Schweiz. This file is part of
+-#  hitobito and licensed under the Affero General Public License version 3
+-#  or later. See the COPYING file at the top-level directory or at
+-#  https://github.com/hitobito/hitobito.
+
+- title ti('label_formats.title')
+
+-if can?(:create_global, LabelFormat)
+  .btn-toolbar
+  %label
+    %input#admin_toggle{ type: 'checkbox', data: {hide: 'admin_labels'} }
+      = I18n.t('label_formats.see_own_labels')
```

Comment:
Soweit ich das verstanden habe, sollte diese Einstellung auch persistiert werden und bei der Anzeige im Export Dropdown Button berücksichtigt werden. Könnte in dem Fall gleich via AJAX auf eine `label_formats#show_global` Action gepostet werden.